tests/test_core.py

```diff
@@ -254,6 +254,34 @@ def test_get_build_config_by_labels(self, openshift):  # noqa
         response = openshift.get_build_config_by_labels(label_selectors)
         assert response['spam'] == 'maps'
 
+    def test_get_build_config_by_labels_filtered(self, openshift):  # noqa
```

Comment:
Could specify the particular reason we're turning off diagnostics: `# noqa:F811`

Reply:
Fixed.